### PR TITLE
Rhv data changes

### PR DESCRIPTION
**Pull request by makentenza**
##### SUMMARY
Move from Gluster to NFS storage domain to reduce the total size requirements. Add new image to contain qcow2 images for VMs and auto-create those VMs during deployment

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
sap-rhvh

### CODE REVIEW

File: ansible/configs/sap-rhvh/default_vars.yml [hunk 1]
```diff
@@ -5,6 +5,12 @@ satellite_activationkey: Summit-2020
 satellite_url: labsat.opentlc.com
 use_content_view: false
 
+# repo_method: satellite
+# satellite_org: Red_Hat_GPTE_Labs
+# satellite_activationkey: SAP_RHV
```
[the remark this answers is not in the document]
Comment (wkulhanek) [reply ansible/configs/sap-rhvh/default_vars.yml:10]: Which means we will probably have to change that one now.... @rut31337

[the remark this answers is not in the document]
Comment (makentenza) [reply ansible/configs/sap-rhvh/default_vars.yml:10]: ok, that's fine, let me move this piece to AgnosticV

[the remark this answers is not in the document]
Comment (makentenza) [reply ansible/configs/sap-rhvh/default_vars.yml]: Info removed

**Comment by makentenza** [comment]
closing this PR and create new one so Sat credentials are not part of it

**Pull request closed without merging**